packages/next-admin/src/components/Form.tsx

```diff
@@ -162,6 +170,10 @@ const Form = ({
     );
   };
 
+  useEffect(() => {
+    setValidation(validationProp);
+  }, [validationProp]);
```

Comment:
Why do we need this state coming from props ?

Reply:
Yeah we actually don't need it since the only place we do this is in the submit handler